service/src/test/java/com/networknt/service/N.java

```diff
@@ -0,0 +1,5 @@
+package com.networknt.service;
+
+public interface N {
```

Comment:
What are these class names? Can they be changed to make their intention at least a bit clear?